docs/pool-upgrade.md

```diff
@@ -0,0 +1,79 @@
+# Pool Upgrade
+
```

Comment:
The provided documentation for the upgrade procedure is really good.
What I think makes sense to add, is a short how-to for Upgrade procedure, that will describe all steps required for an upgrade with the corresponding commands. Probably we can do it in a separate document (`pool-upgrade-howto.md`). The short howto may reference the current doc for details.
For example:
1. New release is scheduled
2. A time (height) at which the new release should be applied is chosen
3. Release checksum is calculated
4. A Trustee sends ProposeUpgrade transaction (<provide CLI command>; specify how to set the binary checksum)
5. Other trustees approve Upgrade (<provide CLI commands>)
6. Node Admins download the upgrade and put it into <to-be-specified> folder
7. Node Admins verify the checksum
8. Wait until update is performed.